src/pages/Login/Login.js

```diff
@@ -2,6 +2,57 @@ import React, { Component } from 'react';
 import './Login.scss';
 
 export class Login extends Component {
+  state = {
+    id: '',
+    password: '',
+    data: [],
+  };
+
+  login = e => {
+    e.preventDefault();
+    fetch('http://10.58.0.244:8000/users/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.id,
+        password: this.state.password,
+      }),
+    })
+      .then(res => res.json())
+      .then(result => {
+        console.log(result);
+        this.setState({ data: result });
+        var token = result.token;
+        localStorage.setItem('token', token);
```

Comment:
- var 는 재선언 / 재할당 모두 가능하기 때문에, 버그의 가능성을 높여 더 이상 사용하지 않습니다.
- 사실 로직을 살펴보면 token 이라는 중간 변수를 생성할 필요가 없어 보입니다!
```suggestion
        localStorage.setItem('token', result.token);
```

Reply:
수정하겠습니다~!